cpp-client/deephaven/examples/concurrent_client/main.cc

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+ */
+#include <exception>
+#include <iostream>
+#include <iomanip>
+#include <memory>
+#include <cstdlib>
+#include <thread>
+#include "deephaven/client/client.h"
+
+using deephaven::client::Client;
+using deephaven::client::TableHandleManager;
+
+void Usage(const char *const argv0, const int exit_status) {
+  std::cerr << "Usage: " << argv0 << "[numthreads [iterations [host:port]]]" << '\n';
+  std::exit(exit_status);
+}
+
+void ThreadFun(std::shared_ptr<Client> clientp, const std::size_t ti) {
+    auto manager = clientp->GetManager();
+    using namespace std::chrono_literals;
+    std::cout << "THREAD START " << ti << std::endl << std::flush;
+    const std::string t1_name = std::string("import deephaven; t1_") + std::to_string(ti);
+    while (true) {
+        manager.RunScript(
+                t1_name + "= deephaven.time_table(\"PT1S\")");
+        std::this_thread::sleep_for(2000ms);
+        auto t1 = manager.FetchTable(std::string("t1_") + std::to_string(ti));
+    }
+    std::cout << "THREAD END " << ti << std::endl << std::flush;
+}
+
+void Run(std::shared_ptr<Client> clientp, const std::size_t nthreads) {
+  std::thread threads[nthreads];
```

Comment:
Making a C-style array with a size determined at runtime is a gcc extension. (Clang supports it too).
The problem is that Visual Studio does *not* support it, so this code will choke on Windows if someone chooses to build it there.
If you don't mind, could you make this a `std::vector<std::thread>` instead?